domain/src/main/java/com/thoughtworks/go/domain/materials/git/GitVersion.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.thoughtworks.go.domain.materials.git;
+
+import de.skuzzle.semantic.Version;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.thoughtworks.go.util.ExceptionUtils.bomb;
+
+public class GitVersion {
+
+    private static final Pattern GIT_VERSION_PATTERN = Pattern.compile(".*\\s+(\\d+.\\d+.\\d+).*");
```

Comment:
This will fail parsing the proper version if given `git version x.y.z (ubuntu 18.04.1)`?

Reply:
This probably works much better, I think: https://github.com/JetBrains/intellij-community/blob/master/plugins/git4idea/src/git4idea/config/GitVersion.java#L64-L65

Reply:
Thanks! I've made the change.